lib/modules/datasource/github-runners/index.ts

```diff
@@ -12,8 +12,10 @@ export class GithubRunnersDatasource extends Datasource {
     ubuntu: [{ version: '22.04' }, { version: '20.04' }, { version: '18.04' }],
     macos: [
       { version: '13' },
+      { version: '13-l' },
```

Comment:
```suggestion
      { version: '13-large' },
```

it seems the labels are wrong?

https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners

Reply:
I was wondering about that too! 🙂